middleware.go

```diff
@@ -36,27 +38,32 @@ func Middleware(serverName string, opts ...Option) func(next http.Handler) http.
 	if cfg.Propagators == nil {
 		cfg.Propagators = otel.GetTextMapPropagator()
 	}
+	if cfg.TraceResponseHeaderKey == "" {
+		cfg.TraceResponseHeaderKey = traceResponseHeaderKey
+	}
```

Comment:
These lines should be deleted since they make, by default, the trace ID will be added to the response header.